### DISCUSSION TITLE
Could not find ionic-team:capacitor-android:3.1.2.

### DISCUSSION THREAD

**Discussion by legiorange** [ionic-team/capacitor#4932] [Q&A] [answered]
When I upgrade capacitor-android 1.1.1 to 3.1.2 it happend:

Executing tasks: [:assembleDebug] in project  
> Task :preBuild UP-TO-DATE
> Task :preDebugBuild UP-TO-DATE
> Task :compileDebugAidl FAILED
> Task :compileDebugRenderscript FAILED
> Task :checkDebugManifest UP-TO-DATE
> Task :generateDebugBuildConfig UP-TO-DATE
> Task :generateDebugResValues UP-TO-DATE
> Task :processDebugManifest
> Task :prepareLintJar UP-TO-DATE
> Task :javaPreCompileDebug FAILED
> Task :mergeDebugConsumerProguardFiles UP-TO-DATE
> Task :mergeDebugShaders UP-TO-DATE
> Task :compileDebugShaders UP-TO-DATE
> Task :generateDebugAssets UP-TO-DATE
> Task :packageDebugAssets UP-TO-DATE
> Task :packageDebugRenderscript NO-SOURCE
> Task :processDebugJavaRes NO-SOURCE
> Task :compileDebugNdk NO-SOURCE
> Task :mergeDebugJniLibFolders UP-TO-DATE
> Task :transformNativeLibsWithMergeJniLibsForDebug UP-TO-DATE
> Task :transformNativeLibsWithSyncJniLibsForDebug UP-TO-DATE

FAILURE: Build completed with 3 failures.

1: Task failed with an exception.
-----------
* What went wrong:
Could not resolve all files for configuration ':debugCompileClasspath'.
> Could not find ionic-team:capacitor-android:3.1.2.
  Searched in the following locations:
    - file:/C:/Users/legio/AppData/Local/Android/Sdk/extras/m2repository/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.pom
    - file:/C:/Users/legio/AppData/Local/Android/Sdk/extras/m2repository/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.jar
    - file:/C:/Users/legio/AppData/Local/Android/Sdk/extras/google/m2repository/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.pom
    - file:/C:/Users/legio/AppData/Local/Android/Sdk/extras/google/m2repository/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.jar
    - file:/C:/Users/legio/AppData/Local/Android/Sdk/extras/android/m2repository/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.pom
    - file:/C:/Users/legio/AppData/Local/Android/Sdk/extras/android/m2repository/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.jar
    - https://dl.google.com/dl/android/maven2/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.pom
    - https://dl.google.com/dl/android/maven2/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.jar
    - https://jcenter.bintray.com/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.pom
    - https://jcenter.bintray.com/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.jar
    - https://repo.maven.apache.org/maven2/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.pom
    - https://repo.maven.apache.org/maven2/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.jar
  Required by:
      project :

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
==============================================================================

2: Task failed with an exception.
-----------
* What went wrong:
Could not resolve all files for configuration ':debugCompileClasspath'.
> Could not find ionic-team:capacitor-android:3.1.2.
  Searched in the following locations:
    - file:/C:/Users/legio/AppData/Local/Android/Sdk/extras/m2repository/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.pom
    - file:/C:/Users/legio/AppData/Local/Android/Sdk/extras/m2repository/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.jar
    - file:/C:/Users/legio/AppData/Local/Android/Sdk/extras/google/m2repository/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.pom
    - file:/C:/Users/legio/AppData/Local/Android/Sdk/extras/google/m2repository/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.jar
    - file:/C:/Users/legio/AppData/Local/Android/Sdk/extras/android/m2repository/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.pom
    - file:/C:/Users/legio/AppData/Local/Android/Sdk/extras/android/m2repository/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.jar
    - https://dl.google.com/dl/android/maven2/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.pom
    - https://dl.google.com/dl/android/maven2/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.jar
    - https://jcenter.bintray.com/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.pom
    - https://jcenter.bintray.com/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.jar
    - https://repo.maven.apache.org/maven2/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.pom
    - https://repo.maven.apache.org/maven2/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.jar
  Required by:
      project :

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
==============================================================================

3: Task failed with an exception.
-----------
* What went wrong:
Could not resolve all files for configuration ':debugCompileClasspath'.
> Could not find ionic-team:capacitor-android:3.1.2.
  Searched in the following locations:
    - file:/C:/Users/legio/AppData/Local/Android/Sdk/extras/m2repository/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.pom
    - file:/C:/Users/legio/AppData/Local/Android/Sdk/extras/m2repository/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.jar
    - file:/C:/Users/legio/AppData/Local/Android/Sdk/extras/google/m2repository/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.pom
    - file:/C:/Users/legio/AppData/Local/Android/Sdk/extras/google/m2repository/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.jar
    - file:/C:/Users/legio/AppData/Local/Android/Sdk/extras/android/m2repository/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.pom
    - file:/C:/Users/legio/AppData/Local/Android/Sdk/extras/android/m2repository/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.jar
    - https://dl.google.com/dl/android/maven2/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.pom
    - https://dl.google.com/dl/android/maven2/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.jar
    - https://jcenter.bintray.com/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.pom
    - https://jcenter.bintray.com/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.jar
    - https://repo.maven.apache.org/maven2/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.pom
    - https://repo.maven.apache.org/maven2/ionic-team/capacitor-android/3.1.2/capacitor-android-3.1.2.jar
  Required by:
      project :

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
==============================================================================

* Get more help at https://help.gradle.org

BUILD FAILED in 4s
15 actionable tasks: 4 executed, 11 up-to-date

How can I download that jar&pom ?

**Comment by jcesarmobile** [comment, marked as answer]
If you were in Capacitor 1, you should read the upgrade guide to Capacitor 2 first and then the guide for Capacitor 3.

In Capacitor 2 upgrade guide there was this step:

https://capacitorjs.com/docs/updating/2-0#remove-maven-repository
